### PR TITLE
Documentation/version1.1

### PR DESCRIPTION
**Pull request by shravan20**
[**CHANGELOGS**]:
- Documentation Updated based on Version 1.1 Release
- README.md Updated

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -126,6 +137,26 @@ You can also provide a file with the list of quotes and a random quote will be g
...
 
+
+---
+
+- ### Category based Quotes
+You can also provide a category to fetch the list of quotes based on certain category and a random quote will be generated for you based on category you specify.
+
+#### Follow the steps to add quotes to category,
+- Add the quote you want in this [file](./customQuotes/category.json) based on the category it falls into.
+- If the category you want doesn't exist, feel free to add the category in this [file](./customQuotes/category.json) and add the quotes.
```
Comment (nishantpersonal) [inline README.md:148]: Add these in the contributing.md file.

File: README.md [hunk 2]
```diff
@@ -126,6 +137,26 @@ You can also provide a file with the list of quotes and a random quote will be g
...
+- Use `quoteCategory=categoryName` as shown below
+  - ```
+    ![Quote](https://github-readme-quotes.herokuapp.com/quote?quoteCategory=motivational)
+    ```
+
+![Quote](https://github-readme-quotes.herokuapp.com/quote?quoteCategory=inspirational)
+
+#### Available Categories
+motivational, fun, life, general, programming, success,
```
Comment (nishantpersonal) [inline README.md:157]: make the categories bullet points for better visualization

**Review by nishantpersonal — CHANGES_REQUESTED** [review]
Please make the following changes

File: README.md (hunk 1 above)
Comment (shravan20) [reply README.md:148]: yes

**Review by shravan20 — COMMENTED** [review]
Changes done

**Pull request merged**